ol-keycloak/ol-spi/src/main/java/edu/mit/keycloak/forms/login/freemarker/models/OpenBean.java

```diff
@@ -0,0 +1,21 @@
+package edu.mit.keycloak.forms.login.freemarker.models;
+
+import java.util.Optional;
+
+import org.keycloak.models.RealmModel;
+
+import edu.mit.keycloak.util.OpenAttribute;
+
+
+public class OpenBean {
```

Comment:
I would prepend "Ol" to this class name in order to more easily distinguish it against OOTB.